docs/contributing-to-airbyte/README.md

```diff
@@ -30,8 +30,8 @@ It's easy to add your own integrations to Airbyte! **Since Airbyte connectors ar
 
 #### **Contributing sources:**
 
-* [In Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source-python/README.md)
-* [Based on Singer Taps in Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source-singer/README.md)
+* [In Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source-python/README.md.hbs)
```

Comment:
```suggestion
You build a Source in any language. Since we frequently build integrations in Python or on top of Singer taps, we've created generator libraries to get you started quickly. 

See (Creating a new integration)[https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations#creating-a-new-integration] for how to get started with creating a new source
```

might want to do something similar in the destination section